README.md

```diff
@@ -166,6 +168,28 @@ GET http://host/endpoint/?field[lte]=value
 ```http request
 GET http://host/endpoint/?type[in][]=one&type[in][]=two
 ```
+- `Tuples` are not allowed in `IN` filter
+
+## `Tuples` in filters
+`Tuples`
+- are important in filters if you have some values, which **must** be send together
+- are composed from two or more values (_`Tuple` of one value is just a value_)
+- items should be in `(` `)` and separated by `,`
+- it is advised NOT to use a `space` between values because of the `URL` specific behaviour
+- for more information about `Tuples` see https://github.com/MortalFlesh/MFCollectionsPHP#immutabletuple 
```

Comment:
The link should be linkable :)

Reply:
it is .. github does it autmatically with links..
https://github.com/MortalFlesh/api-filter/tree/feature/tuple#tuples-in-filters

Reply:
Cool, didn't know that!